pkg/validation/options.go

```diff
@@ -178,6 +182,14 @@ func Validate(o *options.Options) error {
 
 	msgs = append(msgs, validateUpstreams(o.UpstreamServers)...)
 
+	headerSplitRegex := regexp.MustCompile(HeaderSplitRegex)
+	for _, extraHeader := range o.ExtraHeaders {
+		match := headerSplitRegex.FindStringSubmatch(extraHeader)
+		if match == nil {
+			msgs = append(msgs, fmt.Sprintf("error parsing extra header. Expected name=value, got: %s", extraHeader))
+		}
+	}
+
```

Comment:
Would be good to include a test case for this logic

Reply:
Basically out of time in the fiscal year to work on such things. This has taken way more time already then I have to spend on it. I can maybe pick it back up next fiscal if someone else hasn't picked it up.